Documentation/helm-ceph-cluster.md

```diff
@@ -48,22 +48,65 @@ helm install --create-namespace --namespace rook-ceph rook-ceph-cluster \
-| `cephClusterSpec.*`   | Cluster configuration, see below                                     | See below   |
+| Parameter              | Description                                                          | Default     |
+| ---------------------- | -------------------------------------------------------------------- | ----------- |
+| `operatorNamespace`    | Namespace of the Rook Operator                                       | `rook-ceph` |
+| `configOverride`       | Cluster ceph.conf override                                           | <empty>     |
+| `toolbox.enabled`      | Enable Ceph debugging pod deployment. See [toolbox](ceph-toolbox.md) | `false`     |
+| `toolbox.tolerations`  | Toolbox tolerations                                                  | `[]`        |
+| `toolbox.affinity`     | Toolbox affinity                                                     | `{}`        |
+| `monitoring.enabled`   | Enable Prometheus integration, will also create necessary RBAC rules | `false`     |
+| `cephClusterSpec.*`    | Cluster configuration, see below                                     | See below   |
+| `ingress.dashboard`    | Enable an ingress for the ceph-dashboard                             | `{}`        |
+| `cephBlockPools.[*]`   | A list of CephBlockPool configurations to deploy                     | See below   |
+| `cephFileSystems.[*]`  | A list of CephFileSystem configurations to deploy                    | See below   |
+| `cephObjectStores.[*]` | A list of CephObjectStore configurations to deploy                   | See below   |
 
 
 ### Ceph Cluster Spec
 
 The `CephCluster` CRD takes its spec from `cephClusterSpec.*`. This is not an exhaustive list of parameters.
 For the full list, see the [Cluster CRD](ceph-cluster-crd.md) topic.
 
+### Ceph Block Pools
+The `cephBlockPools` array in the values file will define a list of CephBlockPool as described in the table below.
+
+| Parameter                           | Description                                                          | Default          |
+| ----------------------------------- | -------------------------------------------------------------------- | ---------------- |
+| `name`                              | The name of the CephBlockPool                                        | `ceph-blockpool` |
+| `spec`                              | The CephBlockPool spec, see the [CephBlockPool](ceph-pool-crd.md#spec) documentation. | `{}` |
+| `storageClass.enabled`              | Whether a storage class is deployed alongside the CephBlockPool     | `true` |
+| `storageClass.isDefault`            | Whether the storage class will be the default storage class for PVCs which don't request ny particular class to bind to. see the [PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) documentation for details.                                               | `true` |
```

Comment:
I'd like to think the default storage class is a known concept we don't have to describe in the rook docs
```suggestion
| `storageClass.isDefault`            | Whether the storage class will be the default storage class for PVCs. See the [PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) documentation for details.                                               | `true` |
```